locale/en_US/manager.po

```diff
@@ -2090,3 +2090,90 @@ msgstr "Process failed to parse authors"
+msgstr "Signature of the user indicated as a primary contact for this context, it includes full name and a context title"
+
+msgid "emailTemplate.variable.context.contactEmail"
+msgstr "Email address specified in the primary contact field for this context"
+
+msgid "emailTemplate.variable.context.passwordLostUrl"
+msgstr "URL to the lost password page"
+
+msgid "emailTemplate.variable.context.individualSubscriptionUrl"
+msgstr "URL to the user's individual subscription page"
+
+msgid "emailTemplate.variable.context.institutionalSubscriptionUrl"
+msgstr "URL to the institutional subscription page"
+
+msgid "emailTemplate.variable.context.subscriptionContactSignature"
+msgstr "Contact signature derived from a full name, email, phone and mail address specified in the Subscription Policies form"
+
+msgid "emailTemplate.variable.queuedPayment.itemName"
+msgstr "Payment naming"
+
+msgid "emailTemplate.variable.queuedPayment.itemCost"
+msgstr "Payment amount"
+
+msgid "emailTemplate.variable.queuedPayment.itemCurrencyCode"
+msgstr "Currency code for the transaction according to ISO 4217"
+
+msgid "emailTemplate.variable.recipient.userFullName"
+msgstr "Full name of each recipient separated by a comma"
+
+msgid "emailTemplate.variable.recipient.username"
+msgstr "Username of each recipient separated by a comma"
+
+msgid "emailTemplate.variable.recipient.userGivenName"
+msgstr "Given name of each recipient separated by a comma"
+
+msgid "emailTemplate.variable.recipient.reviewDueDate"
+msgstr "Date by which the review is planned to be finished"
+
+msgid "emailTemplate.variable.recipient.responseDueDate"
+msgstr "Final date when response is expected to be received"
+
+msgid "emailTemplate.variable.recipient.submissionReviewUrl"
+msgstr "URL to the submission review assignment"
+
+msgid "emailTemplate.variable.sender.senderName"
+msgstr "Full name of the user sending email"
+
+msgid "emailTemplate.variable.sender.senderEmail"
+msgstr "Sender's email address"
+
+msgid "emailTemplate.variable.sender.senderContactSignature"
+msgstr "Sender's contact signature, which includes a full name, affiliation, and phone number"
+
+msgid "emailTemplate.variable.site.siteTitle"
+msgstr "Title of the website"
+
+msgid "emailTemplate.variable.site.siteContactName"
+msgstr "Full name of the website's primary contact"
+
+msgid "emailTemplate.variable.stageAssignment.editors"
+msgstr "List of editors that are allowed to make decisions on this stage assignment"
+
+msgid "emailTemplate.variable.submission.submissionTitle"
+msgstr "Title of the current publication"
+
+msgid "emailTemplate.variable.submission.submissionId"
+msgstr "Unique ID of the submission"
+
+msgid "emailTemplate.variable.submission.submissionAbstract"
+msgstr "Abstract of the submission"
+
+msgid "emailTemplate.variable.submission.authorString"
+msgstr "Full names of authors separated by a comma"
+
+msgid "emailTemplate.variable.submission.submissionUrl"
+msgstr "Workflow URL of the submission"
```

Comment:
Move the following keys to OJS and change them to:

```
msgid "emailTemplate.variable.context.contextName"
msgstr "The journal's name"

msgid "emailTemplate.variable.context.contextUrl"
msgstr "The URL to the journal's homepage"

msgid "emailTemplate.variable.context.contactName"
msgstr "The name of the journal's primary contact"

msgid "emailTemplate.variable.context.contactEmail"
msgstr "The email address of the journal's primary contact"

msgid "emailTemplate.variable.context.principalContactSignature"
msgstr "The journal's email signature for automated emails"

msgid "emailTemplate.variable.context.individualSubscriptionUrl"
msgstr "The URL to a page where the user can view their individual subscription"

msgid "emailTemplate.variable.context.institutionalSubscriptionUrl"
msgstr "The URL to a page where the user can view their institutional subscription"

msgid "emailTemplate.variable.context.subscriptionContactSignature"
msgstr "An email signature that includes the name, email, phone and mailing address from the journal's Subscription Policies"

msgid "emailTemplate.variable.queuedPayment.itemName"
msgstr "The name of the type of payment"

msgid "emailTemplate.variable.queuedPayment.itemCost"
msgstr "The payment amount"

msgid "emailTemplate.variable.queuedPayment.itemCurrencyCode"
msgstr "The currency of the payment amount, such as USD"

msgid "emailTemplate.variable.site.siteTitle"
msgstr "Name of the website when more than one journal is hosted"
```

Updates for other keys:

```
msgid "emailTemplate.variable.context.passwordLostUrl"
msgstr "The URL to a page where the user can recover a lost password"

msgid "emailTemplate.variable.recipient.userFullName"
msgstr "The full name of the recipient or all recipients"

msgid "emailTemplate.variable.recipient.username"
msgstr "The username of the recipient or all recipients"

msgid "emailTemplate.variable.recipient.userGivenName"
msgstr "The given name of the recipient or all recipients"

msgid "emailTemplate.variable.recipient.responseDueDate"
msgstr "Date when the reviewer should accept or decline the assignment"

msgid "emailTemplate.variable.recipient.reviewDueDate"
msgstr "Date when the review should be completed"

msgid "emailTemplate.variable.recipient.submissionReviewUrl"
msgstr "The URL to the review assignment"

msgid "emailTemplate.variable.sender.senderName"
msgstr "The full name of the sender"

msgid "emailTemplate.variable.sender.senderEmail"
msgstr "The email address of the sender"

msgid "emailTemplate.variable.sender.senderContactSignature"
msgstr "The email signature of the sender"

msgid "emailTemplate.variable.site.siteContactName"
msgstr "The full name of the website's primary contact"

msgid "emailTemplate.variable.stageAssignment.editors"
msgstr "The editors assigned to this submission who can make a decision"

msgid "emailTemplate.variable.submission.submissionTitle"
msgstr "The submission's title"

msgid "emailTemplate.variable.submission.submissionId"
msgstr "The submission's unique ID number"

msgid "emailTemplate.variable.submission.submissionAbstract"
msgstr "The submission's abstract"

msgid "emailTemplate.variable.submission.authorString"
msgstr "The full names of the authors"

msgid "emailTemplate.variable.submission.submissionUrl"
msgstr "The URL to the submission in the editorial backend"
```